content/zh/docs/concepts/configuration/secret.md

```diff
@@ -72,1921 +75,2112 @@ In order to safely use Secrets, take at least the following steps:
+  to expose node-local encryption hardware to a specific Pod. For example, you can schedule
+  trusted Pods onto nodes that provide a Trusted Platform Module, configured out-of-band.
+-->
+- 就身份认证而言，你可以为 X.509 证书实现一个定制的签名者，并使用
+  [CertificateSigningRequest](/zh/docs/reference/access-authn-authz/certificate-signing-requests/)
+  来让该签名者为需要证书的 Pod 发放证书。
+- 你可以使用一个[设备插件](/zh/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/)
+  来将节点本地的加密硬件暴露给特定的 Pod。例如，你可以将可信任的 Pod
+  调度到提供可信平台模块（Trusted Platform Module，TPM）的节点上。
+  这类节点是另行配置的。
 
 <!--
-You can define and use your own Secret type by assigning a non-empty string as the
-`type` value for a Secret object. An empty string is treated as an `Opaque` type.
-Kubernetes doesn't impose any constraints on the type name. However, if you
-are using one of the builtin types, you must meet all the requirements defined
-for that type.
+You can also combine two or more of those options, including the option to use Secret objects themselves.
+
+For example: implement (or deploy) an {{< glossary_tooltip text="operator" term_id="operator-pattern" >}}
+that fetches short-lived session tokens from an external service, and then creates Secrets based
+on those short-lived session tokens. Pods running in your cluster can make use of the session tokens,
+and operator ensures they are valid. This separation means that you can run Pods that are unaware of
+the exact mechanisms for issuing and refreshing those session tokens.
 -->
-通过为 Secret 对象的 `type` 字段设置一个非空的字符串值，你也可以定义并使用自己
-Secret 类型。如果 `type` 值为空字符串，则被视为 `Opaque` 类型。
-Kubernetes 并不对类型的名称作任何限制。不过，如果你要使用内置类型之一，
-则你必须满足为该类型所定义的所有要求。
+你还可以将如上选项的两种或多种进行组合，包括直接使用 Secret 对象本身也是一种选项。
+
+例如：实现（或部署）一个 {{< glossary_tooltip text="operator" term_id="operator-pattern" >}}，
+从外部服务取回生命期很短的会话令牌，之后基于这些生命期很短的会话令牌来创建 Secret。
+运行在集群中的 Pod 可以使用这些会话令牌，而 Operator 则确保这些令牌是合法的。
+这种责权分离意味着你可以运行那些不了解会话令牌如何发放与刷新的确切机制的 Pod。
 
 <!--
-### Opaque secrets
+## Working with Secrets
 
-`Opaque` is the default Secret type if omitted from a Secret configuration file.
-When you create a Secret using `kubectl`, you will use the `generic`
-subcommand to indicate an `Opaque` Secret type. For example, the following
-command creates an empty Secret of type `Opaque`.
+### Creating a Secret
+
+There are several options to create a Secret:
+
+- [create Secret using `kubectl` command](/docs/tasks/configmap-secret/managing-secret-using-kubectl/)
+- [create Secret from config file](/docs/tasks/configmap-secret/managing-secret-using-config-file/)
+- [create Secret using kustomize](/docs/tasks/configmap-secret/managing-secret-using-kustomize/)
 -->
-### Opaque Secret
+## 使用 Secret  {#working-with-secrets}
 
-当 Secret 配置文件中未作显式设定时，默认的 Secret 类型是 `Opaque`。
-当你使用 `kubectl` 来创建一个 Secret 时，你会使用 `generic` 子命令来标明
-要创建的是一个 `Opaque` 类型 Secret。
-例如，下面的命令会创建一个空的 `Opaque` 类型 Secret 对象：
+### 创建 Secret  {#creating-a-secret}」
```

Comment:
```suggestion
### 创建 Secret  {#creating-a-secret}
```

Reply:
Whoops, I really dislike the Mac IME ...